pkg/metrics/metrics.go

```diff
@@ -1100,6 +1111,21 @@ func NewLegacyMetrics() *LegacyMetrics {
 			Help:       "Duration in seconds in how long it took to complete a CNP status update",
 		}, []string{LabelAttempts, LabelOutcome}),
 
+		KubernetesNetworkProgrammingLatency: metric.NewHistogram(metric.HistogramOpts{
+			ConfigName: Namespace + "_" + SubsystemK8s + "_network_programming_duration_seconds",
```

Comment:
Let's at least ensure that the metric is named accurately - this is not currently tracking network programming latency, it is very specific to services.